Makefile.toml

```diff
@@ -94,7 +94,26 @@ description = "Build using no-std"
 command = "cargo"
 args = ["build", "--no-default-features", "--target", "wasm32-unknown-unknown", "--workspace", "--exclude", "miden-mock"]
 
-# --- utilities ----------------------------------------------------------------------------------------
+# --- benchmarking --------------------------------------------------------------------------------
+[tasks.bench-all]
+description = "Run all available transaction benchmarks"
+workspace = false
+command = "cargo"
+args = ["run", "--bin", "benchmarks", "--features", "executable", "all"]
+
+[tasks.bench-tx]
+description = "Run the transaction benchmark with default transaction script and notes"
+workspace = false
+command = "cargo"
+args = ["run", "--bin", "benchmarks", "--features", "executable", "simple"]
+
+[tasks.bench-p2id]
+description = "Run the transaction benchmark which consumes a P2ID note into a basic wallet"
+workspace = false
+command = "cargo"
+args = ["run", "--bin", "benchmarks", "--features", "executable", "p2id"]
```

Comment:
I would have just single `bench-tx` task which would run all benchmarks (i.e., would be the same as `bench-all` task now).